ed25519jni/rust/Cargo.toml

```
@@ -0,0 +1,17 @@
+[package]
+name = "ed25519jni"
+version = "0.0.1"
+authors = ["Douglas Roark <douglas.roark@gemini.com>"]
+license = "MIT OR Apache-2.0"
+publish = false
+edition = "2018"
+
+[dependencies]
+ed25519-zebra = { path = "../../" }
```

Since this crate is published already we probably want to include both the path and the version:
```suggestion
ed25519-zebra = { path = "../../", version = "2.2.0" }
```